frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -75,39 +79,30 @@ const Schedule = () => {
   };
 
   const onEventDrop = ({ start, end, event, resourceId }: any) => {
+    // eslint-disable-next-line no-console
+    console.log('dragged event:', event.title);
+    // eslint-disable-next-line no-console
+    console.log('old resourceId:', event.resourceId);
+    // eslint-disable-next-line no-console
+    console.log('new resourceId:', resourceId);
```

Comment:
get rid of the print statement here?

Reply:
Right, this was something Chris requested I add, but it is unnecessary, so I will comment out the console logs and in the future, if anyone needs to view them, they can uncomment. Later down the line, I will remember to delete the logs entirely.